onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -369,10 +388,9 @@ def test_output_match(self, device: str, dtype: torch.dtype, op):
                     rtol = None
                     atol = None
 
-                # Use torch testing to ensure dtypes and shapes match
```

Comment:
Keep this line?

Reply:
updated in #279

Reply:
sorry, removed by mistake. :)  recalled.